packages/react-keytips/stories/Default.stories.md

```diff
@@ -0,0 +1,4 @@
+By default Keytips are shown for all target elements that have keytip attached via
```

Comment:
Consider incorporating a jsdoc comment to describe the story component, or you could embed the description within `parameters.docs.description.story` itself, instead of using a separate markdown file. A dedicated file for a single paragraph might be unnecessarily elaborate.